db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.1].define(version: 2024_06_05_200000) do
+ActiveRecord::Schema[7.1].define(version: 2024_07_28_123817) do
```

Comment:
Can I leave that to you to make the change? This will also affect the follow-up PR, where the migrations still have the old date. They would then be performed before this migration, which would probably raise an exception since they assume that there already is a vouchers table (they refer to it). So, the dates for the migrations in #671 will need to be changed as well.

Reply:
Sure, done in 02a4e10f656. Will look for it in the second part of the PR as well.